winui/ComboBox/Highlighting.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: post
+title: Highlighting Feature in WinUI Combobox control | Syncfusion
+description: Learn here all about Highlighting Feature in Syncfusion WinUI Combobox control into WinUI application.
+platform: winui
+control: SfComboBox
+documentation: UG
+---
+
+# Highlighting in WinUI ComboBox(SfComboBox)
+
+Highlight matching characters in a suggestion list to pick an item with more clarity. There are two ways to highlight the matching text:
```

Comment:
The highlight matching characters in the suggestion list to pick an item with more clarity. The following are the two ways to highlight the matching text: